README.md

```diff
@@ -32,69 +32,49 @@ println!("{:#?}", recognizer.final_result().multiple().unwrap());
 
 ##### Troubleshooting
 In real-world scenarios, one will use Rust to cross compile a library (e.g. Android and iOS). Therefore, we need both `cdylib` as well as the `staticlib` as crate-type. If you compile as usual with cargo build (e.g.: `cargo build --target aarch64-apple-ios --release`) it will not work, because cargo tries to build the dylib as well. Fortunately, since rust 1.64. there is a new option for [rustc](https://github.com/rust-lang/cargo/issues/10083) in the stable channel. Because of this, the following will work: `cargo rustc --crate-type staticlib --lib --target aarch64-apple-ios --release` 
 
-### Execution (dynamic libraries)
-The libraries also have to be discoverable by the executable at runtime.
-You will have to follow one of the approaches in the same section you chose in [compilation](<#compilation (dynamic libraries)>).
-
-#### Windows and Linux (Recommended)
-For both approaches, you will need to copy the libraries to the root of the executable
-(`target/<cargo profile name>` by default). It is recommended that you use a tool such as 
-[cargo-make](https://sagiegurari.github.io/cargo-make/) to automate moving the libraries
-from another, more practical, directory to the destination during build.
+### Execution
+Dynamically compiled executable must have access to the vosk library at runtime, static prgrams do not.
 
-#### Windows-only
-If you added your libraries to a directory in your `PATH`, no extra steps are needed as long as that is also the case for the target machine.
-
-#### Linux-only
+#### Dynamically compiled executables
+Do either of the following:
 
--   **If you followed option 1 in the [compilation](#linux-only) section:** No extra steps are needed as long as the
-    target machine also has the libraries in one of the mentioned directories.
--   **If you followed option 2:** You will need to add the directory containing the libraries to the
-    `LD_LIBRARY_PATH` environment variable. Note that this directory does not have to be the same added to
-    `LIBRARY_PATH` in the compilation step.
+-   **Recommended:** Copy the libraries to the root of the executable
+    (`target/<cargo profile name>` by default). It is recommended that you use a tool such as
+    [cargo-make](https://sagiegurari.github.io/cargo-make/) to automate moving the libraries
+    from another, more practical, directory to the destination during build.
+-   Make the vosk library accessible system or user-wide:
+    - Windows: Move the libraries to a directory in your `PATH` environment variable.
+    - Linux: Move them to `/usr/local/lib`, `/usr/lib` or set the `LD_LIBRARY_PATH` environment variable to the directory containing the libraries. Note: `LD_LIBRARY_PATH` is not the same as `LIBRARY_PATH` mentioned in the compilation step.
 
-### Execution (static libraries)
 
-#### iOS-only
+#### Statically compiled executables (iOS-only)
```

Comment:
I think this is confusing; the libraries are static, not the executable. It may be better to name the sections something like "Using dynamic libraries" and "Using static libraries".